python/requirements/test-requirements.txt

```diff
@@ -72,7 +72,7 @@ xlrd==2.0.1
 yq==3.2.2
 memray; platform_system != "Windows" and sys_platform != "darwin" and platform_machine != 'aarch64'
 numpy==1.24.4
-ipywidgets==7.7.2
+ipywidgets==8.0.6
```

Comment:
the compiled requirements file also needs to be updated.

Reply:
hmm.. I think the base image that the job is using is failing.

Reply:
is it a chicken and egg problem here, where the base image fails because of an incorrect compiled requirements file, but i cannot generate the compiled requirements file because the base image failure?

Reply:
let me attempt a fix ;)